### ISSUE TITLE
Improvement for #3848

### ISSUE THREAD

**Issue by netzarbeiter** [contao/contao#4122]
### Description

With this great new feature, on the one hand imagick is needed on the server and the formats are given by the underlying imagemagick.

I wrote with @ausi  and we had the idea that it would be nice if the supported file formats would be outputed in the backend e.g. at Source > Source file/Source files.

Displaying of preview images is supported by the following formats: PDF, EPS, PSD, TTF

**Comment by fritzmg** [comment]
imho we should add this info to the "Contao" panel in the Symfony Profiler. In addition we could add:

* supported image formats
* image processing library currently in use

**Comment by netzarbeiter** [comment]
I think this data should not only be available for administrators but also for editors. Therefore I would not hide the formats and image processing library in the symfony profiler.

**Comment by fritzmg** [comment]
I agree actually. I think a new help wizard on the Source/Sources field would be a good place for that?

**Comment by mlwebworker** [comment]
Ich kann mich grob erinnern, dass TYPO3 zu mindestens früher mal ein in meinen Augen pfiffiges Feature hatte - wurde glaube im Rahmen der Installation automatisch gestartet. Dort wurde geprüft und angezeigt, was genau die auf dem Server installierte Bildverarbeitung "kann".

**Comment by frontendschlampe** [comment]
> Ich kann mich grob erinnern, dass TYPO3 zu mindestens früher mal ein in meinen Augen pfiffiges Feature hatte - wurde glaube im Rahmen der Installation automatisch gestartet. Dort wurde geprüft und angezeigt, was genau die auf dem Server installierte Bildverarbeitung "kann".

Haben Sie noch immer https://docs.typo3.org/m/typo3/tutorial-getting-started/main/en-us/Installation/Install.html#scan-environment … sie prüfen während der Installation die kompletten Serveranforderungen. Also im Prinzip wie unser Contao-Check früher. Warum haben wir sowas eigentlich nicht mehr? Ich hatte schon mehrere Serveradmins, die nach solch einem Tool gefragt haben. Die sind von dem Typo3-Check sehr angetan.

**Comment by fritzmg** [comment]
> Warum haben wir sowas eigentlich nicht mehr?

Gibt es grundsätzlich weiterhin im Contao Manager (für die reinen Systemanforderungen).

**Comment by frontendschlampe** [comment]
> Gibt es grundsätzlich weiterhin im Contao Manager (für die reinen Systemanforderungen).

Naja ... aber gerade die Prüfung mit Image-Library wäre ja schon cool. ;-)

**Comment by fritzmg** [comment]
Klar, aber das wären dann zusätzliche Features für den Contao Manager.

**Comment by netzarbeiter** [comment]
Warum diskutiert ihr das nicht in einem separaten Ticket? Hier geht es um eine Information für die Redakteurinnen und Redakteure.
